### PR TITLE
Update BaseStorage

### PR DESCRIPTION
**Pull request by NilsBaumgartner1994**
remove dublicate strings into variables

### CODE REVIEW

File: packages/sdk/src/base/storage/base.ts [hunk 1]
```diff
@@ -4,23 +4,27 @@ export type StorageOptions = {
 	prefix?: string;
 };
 
+const KEY_AUTH_TOKEN: string = 'auth_token';
+const KEY_AUTH_REFRESH_TOKEN: string = 'auth_refresh_token';
+const KEY_AUTH_EXPIRES: string = 'auth_token';
```
Comment (rijkvanzanten) [inline packages/sdk/src/base/storage/base.ts:9]: Lets use a TypeScript Enum:

```suggestion
enum Keys {
  AuthToken = 'auth_token',
  RefreshToken = 'auth_refresh_token',
  Expires = 'auth_expires'
}
```

(also noticed a typo here. KEY_AUTH_EXPIRES is set to "auth_token")

**Review by rijkvanzanten — CHANGES_REQUESTED** [review]

File: packages/sdk/src/base/storage/base.ts (hunk 1 above)
Comment (NilsBaumgartner1994) [reply packages/sdk/src/base/storage/base.ts:9]: Thats actually better +

**Review by NilsBaumgartner1994 — COMMENTED** [review]
Added improvement

File: packages/sdk/src/base/storage/base.ts [hunk 2]
```diff
@@ -4,23 +4,29 @@ export type StorageOptions = {
...
+  RefreshToken = 'auth_refresh_token',
+  Expires = 'auth_expires'
+}
+
 export abstract class BaseStorage extends IStorage {
 	protected prefix: string;
 
 	get auth_token(): string | null {
-		return this.get('auth_token');
+		return this.get(KEY_AUTH_TOKEN);
```
Comment (rijkvanzanten) [inline packages/sdk/src/base/storage/base.ts:17]: Also have to update the uses to actually use the enum 👍🏻

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**